_studio/mfx_lib/vpp/src/mfx_vpp_hw.cpp

```diff
@@ -34,15 +34,15 @@
 #include "mfx_common_int.h"
 #include "mfx_enc_common.h" // UMC::FrameRate
 
+#ifdef MFX_VA_LINUX
```

Comment:
Any reason to change where this is defined?

Reply:
for avoid conflict definition macros "max", "min" (in chrono and cm_mem_copy.h) cm_mem_copy.h should be defined after chrono.
cm_mem_copy.h:
#ifndef max
#define max (a, b) (((a) > (b)) ? (a) : (b))
#endif

Reply:
Can you, please, prepare a change and drop self-define min/max in cm_mem_copy.h. Please, switch to std::min and std::max (use this notation when we call these primitives). This should be a pre-requisite commit. And this change moving headers around should be dropped.